src/Abstractions/NexusMods.Abstractions.Collections/Json/ModSourceType.cs

```diff
@@ -1,13 +1,35 @@
 // ReSharper disable InconsistentNaming
+
+using System.Text.Json.Serialization;
+
 namespace NexusMods.Abstractions.Collections.Json;
 
 /// <summary>
 /// The possible sources of a mod
 /// </summary>
 public enum ModSourceType
 {
-    browse,
-    direct,
-    nexus, 
-    bundle,
+    /// <summary>
+    /// Sourced from Nexus Mods.
+    /// </summary>
+    [JsonStringEnumMemberName("nexus")]
+    NexusMods,
+
+    /// <summary>
+    /// Bundled with the collection archive.
+    /// </summary>
+    [JsonStringEnumMemberName("bundle")]
+    Bundle,
+
+    /// <summary>
+    /// Downloaded externally via an URL.
+    /// </summary>
+    [JsonStringEnumMemberName("Browse")]
```

Comment:
Can you add a quick note that 'browse' usually indicates there's an expectation of user interaction with the URL? And that sometimes it's miscategorised by collection devs.

Might be worth noting, not everyone may always know/remember.